src/vs/workbench/browser/positronNewProjectWizard/utilities/projectNameUtils.ts

```diff
@@ -7,7 +7,20 @@ import { URI } from 'vs/base/common/uri';
 import { localize } from 'vs/nls';
 import { IFileService } from 'vs/platform/files/common/files';
 import { WizardFormattedTextType } from 'vs/workbench/browser/positronNewProjectWizard/components/wizardFormattedText';
-import { IPathService } from 'vs/workbench/services/path/common/pathService';
+
+/**
+ * The maximum length of a project path, which is the full path when joining a parent folder with
+ * the project name.
+ */
+const MAX_LENGTH_PROJECT_PATH = 255;
```

Comment:
Isn't the path length a Windows only limitation? It looks like this will be applied to all platforms.

Reply:
It does look like this maximum was originally added to [address long file paths on Windows](https://github.com/posit-dev/positron/issues/3561), but from some searching, Mac may also have a file name limit of 255 characters, although possibly the full file path length could be up to 1024? I don't have a solid source for this info. We could probably deduce the limit by looking up the file systems used for the versions of Mac we support, and for the Linux distributions as well.

Enforcing the smaller maximum could be beneficial for projects that may be opened on multiple operating systems. We could open a separate issue to look into platform-specific file name/file path length maximum.